### ISSUE TITLE
Latest version of react-md is resolved as 1.17.0 instead of 2.2.2

### ISSUE THREAD

**Issue by rakibulalam** [mlaursen/react-md#948]
./src/style.scss (./node_modules/css-loader/dist/cjs.js??ref--6-oneOf-5-1!./node_modules/postcss-loader/src??postcss!./node_modules/resolve-url-loader??ref--6-oneOf-5-3!./node_modules/sass-loader/dist/cjs.js??ref--6-oneOf-5-4!./src/style.scss)
SassError: Invalid CSS after "(": expected selector, was "(function (global, "
        on line 1 of node_modules/react-md/dist/react-md.js
        from line 1 of /Users/src/style.scss
>> (function (global, factory) {

   ^

```
@import "~react-md/dist/react-md";

@include react-md-utils;
```

**Comment by mlaursen** [comment]
Ahh.. This was my bad since npm dist tags didn't work how I thought. I released `1.17.0` last night under the `latest` tag thinking it would still respect the biggest version number. I guess I'll need to start using a separate `v1` dist tag instead to fix this going forward.

I'll release `2.3.0` after work today even though I was hoping to get Sliders in this release just to fix this error. Until then, you can fix this by installing `2.2.2` (the latest v2 version).

**Comment by rakibulalam** [comment]
Thanks for your replay and waiting for your latest release. Another thing, Missing Grid system In version 2

**Comment by mlaursen** [comment]
Going to close this one now since I actually just released `v2.3.0` which should fix that error.
